employees/templates/employees/common/report_list_navigation_bar.html

```diff
@@ -0,0 +1,80 @@
+{% load rest_framework %}
+
+<div class="row hidden-print">
+  <div class="col-sm-3 hidden-print">
+    {% include "employees/common/month_navigation_bar_with_form.html" %}
+  </div>
+
+  <div class="col-sm-4 hidden-print">
+    {% for filter_name in UI_filter_text %}
+      {% include "employees/common/filter_option.html" %}
+    {% endfor %}
+  </div>
+
+  <div class="col-sm-1 hidden-print">
+    <button class="hidden-print btn btn-default button-without-border" type="button" id="opener_create">
+        <span class="glyphicon glyphicon-edit span-create-report"></span> <font color="#2c77bc"><b>Add new entry</b></font>
+    </button>
+  </div>
+
+  <div class="modal-dialog hidden-print div-text-for-popup" id="dialog_create" title="Create report" align="center">
+      {% for error in errors.non_field_errors %}
+      <h4><font color="red">{{ error }}</font></h4>
+      {% endfor %}
+      <form action="{{ path }}" method="POST">
+          {% csrf_token %}
+          <div class="row">
+            <div class="col-md-5 col-md-offset-0">
+              {% render_field serializer.date style=style %}
```

Comment:
And remember to check everything after rebasing :)

Reply:
Done